specification/hugr.md

```diff
@@ -1746,14 +1734,14 @@ that *v* is lifted to have extension requirement
 R so that it matches the type
 of input to the next iterations of the loop.
 
-$\displaystyle{\frac{\Theta : [R] \textbf{Function}[R](\vec{X}, \vec{Y}) \quad \vec{x} : [R] \vec{X}}{\textbf{call\\_indirect}(\Theta, \vec{x}) : [R] \vec{Y}}}$
+$\displaystyle{\frac{\Theta : [R] \textbf{Function}[R](\vec{X}, \vec{Y}) \quad \vec{x} : [R] \vec{X}}{\textbf{call\_indirect}(\Theta, \vec{x}) : [R] \vec{Y}}}$
```

Comment:
The double backslash is necessary: see https://github.com/CQCL/hugr/blob/spec/cleanups/specification/hugr.md .

Reply:
ah thanks, looks like my local renderer and github disagree